functions/helpFunctions.tsx

```diff
@@ -1,19 +1,42 @@
-export default function cleanHtmlText(content:string) {
-    var cleanHtml = removePTags(content);
+import { TextboxDataType } from '../types/Types';
+
+function cleanHtmlText(content:string) {
+    if (content == undefined) {
+        return
+    }
+    var cleanHtml = removeHtmlTags(content);
     var cleanHtml = cleanHtml.replace(/&#160;/g, ' ');
     var cleanHtml = cleanHtml.replace(/&#230;/g, 'æ'); 
     var cleanHtml = cleanHtml.replace(/&#248;/g, 'ø'); 
     var cleanHtml = cleanHtml.replace(/&#229;/g, 'å'); 
-    var cleanHtml = cleanHtml.replace(/&#62;/g, '>'); 
-
-
+    var cleanHtml = cleanHtml.replace(/&#62;/g, '>');
+    var cleanHtml = cleanHtml.replace(/&#60;/g, ';');
+    var cleanHtml = cleanHtml.replace(/&#39;/g, "'");
+    var cleanHtml = cleanHtml.replace(/&#34;/g, '"');
+    var cleanHtml = cleanHtml.replace(/&#10;/g, '\n');
     return cleanHtml;
 }
 
-function removePTags(htmlString: string) {
+function removeHtmlTags(htmlString: string) {
     var stripedHtml = htmlString.replace(/<p>/g, '');
     var stripedHtml = stripedHtml.replace(/<\/p>/g, '\n');
+    var stripedHtml = stripedHtml.replace(/<strong>/g, '\n');
+    var stripedHtml = stripedHtml.replace(/<\/strong>/g, '\n');
+    var stripedHtml = stripedHtml.replace(/<em>/g, '\n');
+    var stripedHtml = stripedHtml.replace(/<\/em>/g, '\n');
     return stripedHtml;
 }
 
+export function insperaDataToTextboxObject( insperaData: any, questionNumber: number) {
+    const textboxData: TextboxDataType[] = [];
+    if (questionNumber < 1 || questionNumber > insperaData.ext_inspera_candidates[0].result.ext_inspera_questions.length) {
+        return textboxData
```

Comment:
add ";" after return textBoxData

Reply:
For me the editor says "unnecessary semicolon".. I don't get it at all....